src/main/java/org/example/homework9/MobilePhone.java

```diff
@@ -0,0 +1,71 @@
+package org.example.homework9;
+import java.util.ArrayList;
+
+public class MobilePhone {
+    private final ArrayList<Contact> myContacts;
```

Comment:
it should not be final, `myContacts` is modifiable
https://www.baeldung.com/java-final#3-final-fields